### PR TITLE
 GROK-12873: NGL viewer: Initial version of documentation

### PR DESCRIPTION
**Pull request by GennadiyZakharov**
Initial version of documentation for the NGL viewer component

### CODE REVIEW

**Comment by GennadiyZakharov** [comment]
@lbankurova , the failed lint check came not form my changes. These are files added by somebody else in master. Should I fix it?

**Review by lbankurova — CHANGES_REQUESTED** [review]

File: help/visualize/viewers/ngl.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "NGL viewer"
+---
+
+NglViewer is a Datagrok JsViewer
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:5]: Fixed.

File: help/visualize/viewers/ngl.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+You can use Datagrok to explore docking results.
+In this case the NglViewer displays macromolecule structure together with
+structure of ligands
+
+## Creating an NGL viewer
+
+To create a **NGL** viewer, navigate to the **Main Menu**
+and select **Add > Javascript Viewers > NGL**.
+
+Then you need to load the macromolecule file by using the "**Open...**"
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:21]: Fixed.

File: help/visualize/viewers/ngl.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+## Configuring a NGL viewer
+
+You can set the source column for the small molecules
+and customize visualization options.
+To do that, click the **Gear** icon on top of the viewer and use the **Data**
+and **Style** info panes on the **Context Panel**.
+
+You can:
+
+* Change the column with small molecules 3D structures via **Ligand** control.
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:40]: Fixed

File: help/visualize/viewers/ngl.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+* Change macromolecule representation mode via **Representation** control.
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:41]: Fixed.

File: help/visualize/viewers/ngl.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+then the ligand of the current row is displayed in 'green',
+the mouse-over row ligand is displayed in 'light gray',
+and selected rows ligands are displayed in orange.
+
+## Viewer controls
+
+| Action                             | Control                   |
+|------------------------------------|---------------------------|
+| View info for the specific atom    | Hover mouse over the atom |
+| Move the atom to the viewer center | Click on the atom         |
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:60]: Fixed

File: help/visualize/viewers/ngl.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+## Creating an NGL viewer
+
+To create a **NGL** viewer, navigate to the **Main Menu**
+and select **Add > Javascript Viewers > NGL**.
+
+Then you need to load the macromolecule file by using the "**Open...**"
+link in the center of the viewer.
+
+**Note**: User interface for macromolecule loading from a file share
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:24]: Fixed

File: help/visualize/viewers/ngl.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
+---
+title: "NGL viewer"
+---
+
+NglViewer is a Datagrok JsViewer
+derived component based on the [NGL Viewer](nglviewer.org) library developed by Alexander Rose.
+It renders 3D structure of bio macromolecules
+like proteins and DNA/RNA with a variety of representations.
+
+![NGL viewer](ngl-viewer.png)
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:10]: Hist is non-animated image, so I used standard png image, as on the other help pages: 
https://datagrok.ai/help/visualize/viewers/box-plot

I created animated gif showing the representation options.

**Comment by skalkin** [comment]
Images should be 800x500. Or even better, 1600x1000 (with the 2x magnification on non-retina screens)

File: help/visualize/viewers/ngl.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+The NglViewer detects a column of semantic type 'Molecule' in the table
+and displays its data along with the structure.
+
+## Configuring a NGL viewer
+
+You can set the source column for the small molecules
+and customize visualization options.
+To do that, click the **Gear** icon on top of the viewer and use the **Data**
+and **Style** info panes on the **Context Panel**.
```
[the remark this answers is not in the document]
Comment (lbankurova) [reply help/visualize/viewers/ngl.md:32]: Ok to merge but pls fix in master after you're done

**Review by lbankurova — APPROVED** [review]

File: help/visualize/viewers/ngl.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+You can:
+
+* Change the column with small molecules 3D structures via **Ligand** control.
+* Change macromolecule representation mode via **Representation** control.
+
+## Interaction with other viewers
```
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md:41]: Fixed

File: help/visualize/viewers/ngl.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (GennadiyZakharov) [reply help/visualize/viewers/ngl.md]: Aimated GIFs created.

**Pull request merged**